packages/terra-consumer-nav/src/components/nav-items/NavItems.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import NavToggler from '../nav-toggler/NavToggler';
+import styles from './NavItems.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  navItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      slug: PropTypes.string,
```

Comment:
A cleaner approach is to have a `NavItem` component. Build an array of those and pass that array to the `NavItems` component. You can then ensure that `NavItems` takes an instance of `NavItem`